DatadogRUM/Sources/RUMMonitor/Scopes/RUMViewScope.swift

```diff
@@ -532,7 +532,6 @@ internal class RUMViewScope: RUMScope, RUMContextProvider {
 
     private func sendErrorEvent(on command: RUMAddCurrentViewErrorCommand, context: DatadogContext, writer: Writer) {
         errorsCount += 1
-        attributes.merge(rumCommandAttributes: command.attributes)
```

Comment:
Question: 
Can we consider still merging the Error and View attributes together, while not modifying the View attributes?

This will reduce the scope of the change in this PR. 
Also the Browser SDK does propagate all RUM View attributes to children events, so we might move in this direction as well for mobile SDKs. 

But I'm not against aligning all mobile SDKs and events.

Reply:
EDIT: Browser SDK does propagate all global attributes to children events, there is no RUM View attributes. 
So this change will actually bring all SDKs closer 👍